src/app/components/turnos/gestor-agendas/pipes/botonesAgendaGeneral.pipe.ts

```diff
@@ -0,0 +1,116 @@
+import { Auth } from '@andes/auth';
+import { Pipe, PipeTransform } from '@angular/core';
+import { IAgenda } from '../../../../interfaces/turnos/IAgenda';
+
+@Pipe({
+    name: 'botonesAgendaGeneral'
+})
+export class BotonesAgendaGeneralPipe implements PipeTransform {
+    constructor(private auth: Auth) { }
+    transform(agendas: IAgenda[]): any {
+        const botones = {
+            publicar: false,
+            borrar: false,
+            disponible: false,
+            reanudar: false,
+            imprimir: false,
+            imprimirCarpetas: false,
+            nota: false,
+            pausar: false
+        };
+
+        const puedeImprimir = this.auth.getPermissions('turnos:agenda:puedeImprimir:').length > 0;
```

Comment:
Idem al comentario anterior: `this.auth.check` x `this.auth.getPermissions`

Reply:
Corregido